src/gpu/intel/ocl/gemm/gemm_with_post_ops.hpp

```diff
@@ -61,11 +62,15 @@ struct gemm_with_post_ops_t : public gpu_gemm_t {
                         &threads_per_eu)) {
             CHECK(attr.set_gpu_attr(gpu_primitive_attr_t(threads_per_eu)));
         }
+        post_process_kernels_.resize(2);
         compute::kernel_ctx_t kernel_ctx(&attr);
         ret_status = pd()->init_kernel_ctx(kernel_ctx);
         CHECK(ret_status);
         ret_status = create_kernel(
-                engine, &post_process_kernel_, "gemm_post_ops", kernel_ctx);
+                engine, &post_process_kernels_[0], "gemm_post_ops", kernel_ctx);
+        if (pd()->subbyte_pack_)
```

Comment:
[nitpick]
If it's just a couple kernels, maybe it's worth creating a var for each one?